packaging/ubuntu-16.04/rules

```diff
@@ -207,7 +212,7 @@ endif
 	(cd c-vendor/squashfuse && mkdir -p autom4te.cache && ./autogen.sh --disable-demo && ./configure --disable-demo && make && mv squashfuse_ll snapfuse)
 
 override_dh_auto_test:
-	dh_auto_test -- $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS)
+	dh_auto_test -- -mod vendor $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS) $(DH_GOPKG)/...
```

Comment:
it probably doesn't matter, but someplaces we use `-mod vendor` and some places we use `-mod=vendor`, should we be consistent?